recipes/jaws/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2.3" %}
+
+package:
+  name: jaws
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaws/jaws/archive/{{ version }}.tar.gz
+  sha256: b8cd6e76f01a3786d32880c757b02aa82651eb9fffeb0bc8aa155dac02212b38
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas
+    - numpy >=1.11
+    - xarray
+    - pytz
+    - netcdf4
+    - matplotlib
+    - netcdf4
+  run:
+    - python
+    - pandas
+    - numpy
+    - xarray
+    - pytz
+    - netcdf4
+    - matplotlib
+    - netcdf4
+
+test:
+  commands:
+    - 
+
+about:
+  home: https://github.com/jaws/jaws
+  license: Apache, Version 2.0
```

Comment:
Needs a license family and a license file

Reply:
Added license family and license file.